avm/res/kubernetes/connected-clusters/main.json

```diff
@@ -0,0 +1,188 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "languageVersion": "2.0",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "_generator": {
+      "name": "bicep",
+      "version": "0.32.4.45862",
```

Comment:
Not mandatory but just as a heads up that there's a most recent bicep version available